content/rdi/installation/install-k8s.md

```diff
@@ -113,50 +113,79 @@ Run `redis-di status` to check the status of the installation.
 
 ## Install RDI CLI on Kubernetes Cluster
 
-### Add CLI pod
+### Add CLI deployment
 
 ```bash
-cat << EOF > /tmp/redis-di-cli-pod.yml
-apiVersion: v1
-kind: Pod
+cat << EOF > /tmp/redis-di-cli-deployment.yml
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: redis-di-cli
-  labels:
-    app: redis-di-cli
 spec:
-  containers:
-    - name: redis-di-cli
-      image: docker.io/redislabs/redis-di-cli
-      volumeMounts:
-      - name: config-volume
-        mountPath: /app
-      - name: jobs-volume
-        mountPath: /app/jobs
-  volumes:
-    - name: config-volume
-      configMap:
-        name: redis-di-config
-        optional: true
-    - name: jobs-volume
-      configMap:
-        name: redis-di-jobs
-        optional: true
+  replicas: 1
+  selector:
+    matchLabels:
+      app: redis-di-cli
+  template:
+    metadata:
+      labels:
+        app: redis-di-cli
+    spec:
+      containers:
+        - name: redis-di-cli
+          image: docker.io/redislabs/redis-di-cli:latest
+          volumeMounts:
+          - name: config-volume
+            mountPath: /app
+          - name: jobs-volume
+            mountPath: /app/jobs
+      volumes:
+        - name: config-volume
+          configMap:
+            name: redis-di-config
+            optional: true
+        - name: jobs-volume
+          configMap:
+            name: redis-di-jobs
+            optional: true
 EOF
-kubectl apply -f /tmp/redis-di-cli-pod.yml
+kubectl apply -f /tmp/redis-di-cli-deployment.yml           
+```
+
+After creation of the deployment, the RDI CLI will be available as a pod in the cluster. It can be scaled down to 0 replicas when not in use or during maintenance. Use the following commands to scale down/up the deployment:
+
+```bash
+kubectl scale deployment.apps/redis-di-cli --replicas=0
+kubectl scale deployment.apps/redis-di-cli --replicas=1
+```
+
+> Note: Before scaling down the deployment it is recommended to save the redis-di context for future use. To do this, run the following command:
+>
+> ```bash
+> cat /root/.redis-di
+> ```
+>
+> Copy the output of the command and save it in a local file. To use the context in the future, simply create the file `/root/.redis-di` and paste the content saved before. This is useful when the RDI CLI deployment is scaled up or upgraded to a new image.
```

Comment:
Not sure it's understood that this is part of the note section, maybe consider changing it to a sub section instead of a note.

Reply:
It is indeed a part of the note section when you look at the preview:
![image](https://github.com/RedisLabs/redislabs-docs/assets/98769322/6b197153-b405-4853-9e8d-4b12fbb254b9)
I wouldn't change it to a separate subsection since this is a comment to the scaling statement above.

Reply:
Yes, it is already like that.